code/datums/emotes.dm

```diff
@@ -179,6 +179,15 @@
 					to_chat(user, span_notice("You cannot [key] while stunned."))
 					return FALSE
 
+		if(flags_emote & EMOTE_ARMS_CHECK)
+			if(ishuman(user))
```

Comment:
that means a non human can snap non-existing fingers :/

Reply:
Fuck non humans

Reply:
fixed

Reply:
Tested too!